src/cljs/rems/util.cljs

```diff
@@ -67,17 +70,32 @@
       (when handler
         (handler err)))))
 
+(defn- append-handler [old-handler new-handler]
+  (fn [response]
+    (try
+      (when old-handler
+        (old-handler response))
+      (finally
+       (new-handler response)))))
+
 (defn fetch
   "Fetches data from the given url with optional map of options like #'ajax.core/GET.
 
   Has sensible defaults with error handler, JSON and keywords.
 
-  Additionally calls event hooks."
+  Additionally calls event hooks.
+
+  Returns a promise, but it's okay to ignore it if you prefer using the event hooks."
```

Comment:
This is misleading, hooks refers to the window.rems.hooks below. You probably meant

> it's okay to ignore it if you prefer using `:handler` and/or `:error-handler`

Reply:
Oh, that's what the previous comment about hooks meant. 🤣